### PR TITLE
[ADVISOR-2339] Task status and results page

### PR DESCRIPTION
**Pull request by johnsonm325**

### CODE REVIEW

File: src/SmartComponents/CompletedTaskDetails/Columns.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    {display_name}
+  </Link>
+);
+
+SystemNameCell.propTypes = {
+  id: propTypes.string,
+  display_name: propTypes.string,
+};
+
+export const System = {
```
Comment (mkholjuraev) [inline src/SmartComponents/CompletedTaskDetails/Columns.js:21]: I think renaming these column names to have 'Column' string appended at the end would make the code more readable (for example: System => SystemColumn).

**Review by mkholjuraev — COMMENTED** [review]
I could not run the PR locally as I am not behind VPN and I can only use prod env here. The code looks good to me, I have left some comments that I thought worth.

File: src/SmartComponents/CompletedTaskDetails/Columns.js (hunk 1 above)
Comment (johnsonm325) [reply src/SmartComponents/CompletedTaskDetails/Columns.js:21]: I'm not sure I understand what you mean here.

File: src/Utilities/__tests__/helpers.tests.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import items from '../hooks/useTableTools/__tests__/__fixtures__/items.fixtures';
+
+describe('uniq', () => {
+  it('should deduplicate items', () => {
+    let result = uniq(['a', 'b', 'b', 'c']);
+    expect(result.sort()).toEqual(['a', 'b', 'c']);
+  });
+});
+
+describe('orderArrayByProp', () => {
```
[the remark this answers is not in the document]
Comment (johnsonm325) [reply src/Utilities/__tests__/helpers.tests.js:11]: Maybe you can show me this time how the each function would work with snapshot testing here? I'm not sure I understand how it works. Might be quicker to learn from you than from docs. :)

File: src/SmartComponents/CompletedTaskDetails/Columns.js (hunk 1 above)
Comment (mkholjuraev) [reply src/SmartComponents/CompletedTaskDetails/Columns.js:21]: We usually name React components in this manner with capital letters. When I saw the Systems object on line #21, I immediately thought that it should be component. Thus, I suggested to rename it to SystemColumn, to easly say that this is not React Component.

Or you can easly  make it lower case.

File: src/Utilities/__tests__/helpers.tests.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mkholjuraev) [reply src/Utilities/__tests__/helpers.tests.js:11]: Yes, of course. Feel free to ping me anytime please.

**Comment by johnsonm325** [comment]
@mkholjuraev Okay, I'm going to hold off on the testing portion and merge this with your suggested changes. It doesn't _need_ approval and I need to work quick. Thanks for taking time to look at this.

File: src/SmartComponents/CompletedTaskDetails/Columns.js (hunk 1 above)
Comment (johnsonm325) [reply src/SmartComponents/CompletedTaskDetails/Columns.js]: ahh, I see. The `System => SystemColumn` threw me off. I thought you wanted me to set up the function to look like that and I didn't understand. :)

**Pull request merged**